lld/COFF/InputFiles.cpp

```diff
@@ -1019,9 +1020,16 @@ void ImportFile::parse() {
 
   // Read names and create an __imp_ symbol.
   StringRef buf = mb.getBuffer().substr(sizeof(*hdr));
-  StringRef name = saver().save(buf.split('\0').first);
+  StringRef nameBuf = buf.split('\0').first, name;
```

Comment:
Stylistically, I dislike the `type var = initialization, otherVar;` style - if there's an initialization of one variable (and a nontrivial one in this case) I find it more readable to have the other variable on a separate line, or perhaps reordered so that the pure name without an assignment first.